official/utils/testing/reference_data.py

```diff
@@ -261,12 +261,19 @@ def _evaluate_test_case(self, name, graph, ops_to_eval, correctness_function):
             .format(differences, tf_version_comparison))
 
       eval_results = [op.eval() for op in ops_to_eval]
+      print("eval_resuls:{}".format(eval_results))
       if correctness_function is not None:
         results = correctness_function(*eval_results)
         result_json = os.path.join(data_dir, "results.json")
+        print("results:{}".format(results))
+        print("data_dir:{}".format(data_dir))
         with tf.io.gfile.GFile(result_json, "r") as f:
           expected_results = json.load(f)
-        self.assertAllClose(results, expected_results)
+          print("expected_results:{}".format(expected_results))
+        self.assertAllClose(results,
+                            expected_results)
+                            # rtol=1e-05,
```

Comment:
what about these commented tolerances?

Reply:
I removed them, it defaults to 1e-06 according to the docs and that is fine.